tools/squashfs4/Makefile

```diff
@@ -16,26 +16,89 @@ PKG_SOURCE_URL:=https://github.com/plougher/squashfs-tools
 PKG_SOURCE_DATE:=2022-03-17
 PKG_SOURCE_VERSION:=afdd63fc386919b4aa40d573b0a6069414d14317
 PKG_MIRROR_HASH:=59c294e00ddfc1f4bda658c7567584ce35fdd479c413ebdb1376981b27737be1
+PKG_CONFIG_DEPENDS:=	CONFIG_TOOLS_SQUASHFS_GZIP	\
+			CONFIG_TOOLS_SQUASHFS_LZ4	\
+			CONFIG_TOOLS_SQUASHFS_LZO	\
+			CONFIG_TOOLS_SQUASHFS_XATTR	\
+			CONFIG_TOOLS_SQUASHFS_XZ	\
+			CONFIG_TOOLS_SQUASHFS_ZSTD
```

Comment:
I do not like if this gets rebuild every time we change the default squashfs compression.
If you anyway have all the dependencies to build support for all these compression methods, I would prefer if you always activate them.

Reply:
I figured the compile-time or size-minimization crew would complain if every method (and dep thereof) were pulled in for nothing.  Maybe nobody cares in the tools/toolchain.  It does get a whole lot simpler without tracking and just building a does-all version.  Adjustment pending...

Reply:
Now it just builds with all features.